src/openrct2/object/Object.h

```diff
@@ -26,6 +26,8 @@ using ObjectEntryIndex = uint16_t;
 constexpr const ObjectEntryIndex OBJECT_ENTRY_INDEX_NULL = std::numeric_limits<ObjectEntryIndex>::max();
 struct ObjectRepositoryItem;
 
+using Version = std::tuple<uint16_t, uint16_t, uint16_t>;
```

Comment:
A flexible container like `std::vector` allows for more flexibility with version strings.

Reply:
That would be bad idea you would be allocating like crazy this needs to be very light weight as it will be used in a lot of locations